torch/lib/THC/generic/THCTensorMathReduce.cu

```diff
@@ -146,10 +146,18 @@ THCTensor_(varall)(THCState *state, THCTensor *self)
     THArgCheck(false, 1, CUTORCH_DIM_WARNING);
   }
 
-  val = THCNumerics<accreal>::div(
-    val,
-    ScalarConvert<ptrdiff_t, accreal>::to(THCTensor_(nElement)(state, self) - 1)
-  );
+  if (flag) {
+    val = THCNumerics<accreal>::div(
+      val,
+      ScalarConvert<ptrdiff_t, accreal>::to(THCTensor_(nElement)(state, self))
+    );
+  }
+  else {
+    val = THCNumerics<accreal>::div(
+      val,
+      ScalarConvert<ptrdiff_t, accreal>::to(THCTensor_(nElement)(state, self) - 1)
+    );
+  }
```

Comment:
You can use the ternary operator to clean that up just like above. It's repeated unnecessarily